geode-apis-compatible-with-redis/src/distributedTest/java/org/apache/geode/redis/internal/executor/set/SremDUnitTest.java

```diff
@@ -218,20 +268,17 @@ public void shouldDistributeDataAmongMultipleServers_givenMultipleClientsOnSameS
     allMembers.addAll(members1);
     allMembers.addAll(members2);
 
-    jedis3.sadd(key, allMembers.toArray(new String[] {}));
+    jedis.sadd(key, allMembers.toArray(new String[] {}));
 
     new ConcurrentLoopingThreads(SET_SIZE,
-        (i) -> jedis1.srem(key, members1.get(i)),
-        (i) -> jedis1B.srem(key, members1.get(i)),
-        (i) -> jedis2.srem(key, members2.get(i)),
-        (i) -> jedis2B.srem(key, members2.get(i))).run();
+        (i) -> jedis.srem(key, members1.get(i)),
+        (i) -> jedis2.srem(key, members2.get(i))).run();
```

Comment:
This is the same test as `shouldDistributeDataAmongCluster_givenMultipleClientsOnDifferentServer_removingDifferentDataFromSameSetConcurrently`